static/js/apps/eval_retrieval_generation/sxs/left_right_picker.ts

```diff
@@ -35,7 +37,8 @@ export function getLeftAndRight(
   const inputs =
     `${docInfoA.doc.spreadsheetId} ${docInfoB.doc.spreadsheetId}` +
     ` ${sessionId} ${queryId}`;
-  if (sdbm(inputs) % 2 === 0) {
+  const seededRand = new Srand(sdbm(md5(inputs)));
+  if (Math.floor(seededRand.random() * 2) === 0) {
```

Comment:
Add a comment why we do this?

Would it be simpler to use `intInRange` method?

Also, wonder if there was a specific reason you chose this vs. others (like say https://npm.io/package/pure-rand released more recently and has more downloads)?

Reply:
I just used what I could find, happy to switch to a package with better support! Will add a comment too.